File: Examples/VisualBasic/NoteBook/ConvertToPDF.vb
Imports System.IO
Imports AsposeNote = Aspose.Note
Imports System.Collections.Generic

Namespace NoteBook
    Public Class ConvertToPDF
        Public Shared Sub Run()
            ' ExStart:ConvertToPDF
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_NoteBook()

            ' Load a OneNote Notebook
            Dim notebook = New AsposeNote.Notebook(dataDir & Convert.ToString("Notizbuch öffnen.onetoc2"))

            dataDir = dataDir & Convert.ToString("ConvertToPDF_out_.pdf")

            ' Save the Notebook
            notebook.Save(dataDir)

            ' ExEnd:ConvertToImage
            Console.WriteLine(Convert.ToString(vbLf & "NoteBook document converted to pdf successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace

File: Examples/CSharp/NoteBook/ConvertToPDF.cs
using System.IO;
using Aspose.Note;
using System;
using System.Collections.Generic;

namespace Aspose.Note.Examples.CSharp.WorkingWithNoteBook
{
    public class ConvertToPDF
    {
        public static void Run()
        {
            // ExStart:ConvertToPDF
            // ExFor:Notebook
            // ExSummary:Shows how to save notebook in pdf format.

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_NoteBook();

            // Load a OneNote Notebook
            var notebook = new Notebook(dataDir + "Notizbuch öffnen.onetoc2");

            dataDir = dataDir + "ConvertToPDF_out.pdf";

            // Save the Notebook
            notebook.Save(dataDir);

            // ExEnd:ConvertToPDF
            Console.WriteLine("\nNoteBook document converted to pdf successfully.\nFile saved at " + dataDir);
        }
    }
}